cloud.foundry.cli/src/test/java/cloud/foundry/cli/crosscutting/mapping/CfOperationsCreatorTest.java

```diff
@@ -18,6 +18,7 @@
 
```

Comment:
Using [PowerMock](https://github.com/powermock/powermock/wiki/Mockito) additionaly to Mockito we could be able to mock the static method System.getenv and test the newly implemented functionality with new unit tests.

Reply:
Its not possible by our current configuration. 
Because PowerMock would work with JUnit4, but not for JUnit5. 

If I would migrate PowerMock, I have to configure our JUnit5 into JUnit4, what I don't want to do.

Reply:
here: 
https://github.com/powermock/powermock/issues/929

Reply:
After a bit of research I found out that environmet variables for tests can be set using gradle (see [here](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.testing.Test.html#org.gradle.api.tasks.testing.Test:environment)). Would that be a sensible way to test it?

Reply:
Looks interessting. 
But still, I would either prefer to: 
- keep it as it is 
- migrate to JUnit4 
- search for another library

One problem of this solution is,
that I am not sure, if I can discover multiple cases with it like: 
test {
  environment "VAR", "val"
}

I have "some value"
or 
I don't have a value and the environment variable shall return "null". 


Also I don't like the way to mock it with gradle. 
Because someone will be confused, that he has to look into two places. 
The best would be, to mock the things in the Unit-Test and thats it. 

Another aspect is, that gradle is just an build application and it should be also possible, to use another build application like maven. 
I am not sure, if it is also provided in maven like it is in gradle (Vendor-lock-in).

Even if we would use it on this way, 
it would be just a "workaround", because Mockito is not able to mock it. 
We will maybe have the same problem in another feature in future. 


So I would still either keep it, or create another ticket to migrate the tests in JUnit4, because this would be more effort and extend the current scope.